composer.json

```diff
@@ -18,15 +18,17 @@
 	"require": {
 		"php": "^5.3.9|~7.0",
 		"doctrine/doctrine-bundle": "~1.3",
-		"symfony/framework-bundle": "~2.7|~3.0"
+		"symfony/framework-bundle": "~2.7|~3.0|~4.0",
+		"symfony/form": "~2.7|~3.0|~4.0",
+		"symfony/twig-bundle": "~2.7|~3.0|~4.0",
+		"symfony/asset": "~2.7|~3.0|~4.0",
+		"symfony/translation": "~2.7|~3.0|~4.0"
```

Comment:
These four dependencies (form, twig-bundle, asset, translation) aren't necessarily needed, only if the built-in form is used.